<launch>

  <param name="naio01_server_port" type="int" value="5555" />

  <node name="Core" pkg="oz440_gazebo" type="Core"/>

  <!-- We resume the logic in empty_world.launch, changing only the name of the world to be launched -->
  <include file="$(find gazebo_ros)/launch/empty_world.launch">
    <arg name="world_name" value="$(find oz440_gazebo)/worlds/my_world.world"/>
  </include>

    <param name="robot_description" command="$(find xacro)/xacro '$(find oz440_description)/urdf/oz440.xacro'" />

	<!-- Spawn a robot into Gazebo -->
	<node name="spawn_urdf" pkg="gazebo_ros" type="spawn_model" output="screen" args="-param robot_description -b -urdf -x 0 -y 0 -z 0.4 -model oz440" />

	<!-- ros_control launch file -->
    <include file="$(find oz440_control)/launch/oz440_control.launch" />
 

</launch>
